src/Component/Modal/ApplicationInfo/ApplicationInfo.tsx

```diff
@@ -40,7 +40,7 @@ export const ApplicationInfo: React.FC<ApplicationInfoProps> = ({
   return (
     <Modal
       className="application-info-modal"
-      title={t('ApplicationInfoModal.title')}
+      title={<div className="application-title">{t('ApplicationInfoModal.clientAbout')}</div>}
```

Comment:
you don't really need the extra className here since the modal already has the class `application-info-modal`

in your css rule you could just use `.application-info-modal .ant-modal-title` to select the title